activerecord/lib/active_record/query_logs.rb

```diff
@@ -6,7 +6,31 @@
 module ActiveRecord
   # = Active Record Query Logs
   #
-  # Automatically tag SQL queries with runtime information.
+  # Automatically append comments to SQL queries with runtime information tags. This can be used to trace troublesome
+  # statements found using database tooling back to the area of the application code where these statements were
+  # generated by Active Record.
+  #
+  # Query logs can be enabled via Rails configuration in <tt>config/application.rb</tt> or any environment initializer:
+  #
+  #     config.active_record.query_log_tags_enabled = true
+  #
+  # or directly:
+  #
+  #     ActiveRecord.query_log_tags_enabled = true
+  #
+  # By default the name of the application, the name and action of the controller, or the name of the job are logged
+  # using the {SQLCommenter}[https://open-telemetry.github.io/opentelemetry-sqlcommenter/] format.
```

Comment:
Is it worth documenting the `app.config.active_record.query_log_tags_format` option?

Note that the default is actually dependent on the Rails version, see https://github.com/rails/rails/blob/main/guides/source/configuring.md#configactive_recordquery_log_tags_format

Reply:
Done!